modules/serverless-logic-job-service-custom-execution.adoc

```diff
@@ -0,0 +1,104 @@
+
+.Procedure
+
+. Identify the image to use depending on the persistence mechanism, and see the required configuration parameters specific for that image.
+. Identify if you need the Eventing API, and check the required configuration parameters for it.
+. Ensure that the project containing your workflows is set up with the correct Job service Quarkus extension.
+. When running the image, pass the necessary configurations through environment variables or as system properties using Java-like names.
+
+[id="serverless-logic-job-service-custom-execution-env-variables_{context}"]
+== Custom execution using environment variables
+
+To configure the image using environment variables, pass one environment variable for each parameter by running the following command:
+
+.Example of Job service image configuration
+[source,terminal]
+----
+$ oc new-app registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:latest \
+  --name=jobs-service-postgresql \
+  -e QUARKUS_DATASOURCE_USERNAME=postgres \
+  -e VARIABLE_NAME=value
+----
+
+.Example of Job service image configuration for OpenShift execution
+[source,yaml]
+----
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: jobs-service-postgresql
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jobs-service
+  template:
+    metadata:
+      labels:
+        app: jobs-service
+    spec:
+      containers:
+        - name: jobs-service-postgresql
+          image: registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:latest
+          imagePullPolicy: Always
+          ports:
+            - containerPort: 8080
+              name: http
+              protocol: TCP
+          env:
+            # Set the image parameters as environment variables in the container definition.
+            - name: KUBERNETES_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+            - name: QUARKUS_DATASOURCE_USERNAME
+              value: postgres
+            - name: QUARKUS_DATASOURCE_PASSWORD
+              value: pass
+            - name: QUARKUS_DATASOURCE_JDBC_URL
+              value: jdbc:postgresql://timeouts-showcase-database:5432/postgres?currentSchema=jobs-service
+            - name: QUARKUS_DATASOURCE_REACTIVE_URL
+              value: postgresql://timeouts-showcase-database:5432/postgres?search_path=jobs-service
+----
+
+[id="serverless-logic-job-service-custom-execution-system-properties_{context}"]
+== Custom execution using system properties with java like names
+
+To configure the image by using system properties you must pass one property per parameter, however, in this case, all these properties are passed as part of a single environment with the name `JAVA_OPTIONS` by running the following command:
+
+.Example of Job service image configuration
+[source,terminal]
+----
+$ oc new-app registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:latest \
```

Comment:
Shouldn't it be with the tag version instead of latest? Following @wmedvede's comment, it's better to use a variable {ServerlessLogicProductVersion}
```suggestion
$ oc new-app registry.redhat.io/openshift-serverless-1/logic-jobs-service-postgresql-rhel8:1.34.0 \
```

Reply:
Added